src/components/Form/index.tsx

```diff
@@ -0,0 +1,138 @@
+import {
+  Dispatch,
+  FC,
+  SetStateAction,
+  useState,
+} from 'react';
+import usersFromServer from '../../api/users';
+import { TodoItem } from '../../types/todo';
+
+const Form: FC<{ updateState: Dispatch<SetStateAction<TodoItem[]>> }> = ({
```

Comment:
better to use interface for type Props
![image](https://github.com/mate-academy/react_add-todo-form/assets/96265844/36bc0977-df10-46f5-9e52-36d6580d7b3c)